app/components/MessageGradientTextLoading.tsx

```diff
@@ -0,0 +1,20 @@
+import { PropsWithChildren } from 'react';
+
+interface MessageGradientTextLoadingProps extends PropsWithChildren {
+  className?: string;
+}
+
+export const MessageGradientTextLoading: React.FC<MessageGradientTextLoadingProps> = ({
```

Comment:
Why a new component just for elements with style?

Reply:
we might need it in more places, but yeah, we can do it inline right now.